zh/about.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: About
+permalink: /zh/about/
+redirect_from: /about
+---
+
+Bitcoin Operations Technology Group（简称 Optech）致力于将最好的开源技术和技术手段引入使用比特币的企业，以降低成本并改善客户体验。
+
+该组织的初期重点是与其成员组织合作，减少交易大小并尽量减少随后的交易费用增加的影响。我们提供[研讨会][workshops]、[文档][scaling book]、[每周通讯][weekly newsletters]、[原创研究][dashboard]、[案例研究和公告][blog]、[播客][podcast]，并帮助促进企业与开源社区之间的关系。
+
+[scaling book]: https://github.com/bitcoinops/scaling-book
+[workshops]: /zh/workshops
+[weekly newsletters]: /zh/newsletters/
+[dashboard]: https://dashboard.bitcoinops.org/
+[blog]: /zh/blog/
+[podcast]: /zh/podcast/
```

Comment:
I dont think we plan on getting zh version of the podcast or transcriptions so I think the en version is fine here

```suggestion
[podcast]: /en/podcast/
```

Reply:
> I dont think we plan on getting zh version of the podcast or transcriptions so I think the en version is fine here

So others are kept and `schoor-taproot-workshop` are also in EN right?

Reply:
@bittomhan up to you. The videos and notebooks in the workshop are all en so perhaps it is not worth translating that homepage?

Reply:
> @bittomhan up to you. The videos and notebooks in the workshop are all en so perhaps it is not worth translating that homepage?

OK, I will keep all that in EN.